pkg/proc/amd64util/xsave_amd64.go

```diff
@@ -28,3 +28,28 @@ func AMD64XstateMaxSize() int {
 	})
 	return xstateMaxSize
 }
+
+var xstateZMMHi256Offset int
+var loadXstateZMMHi256OffsetOnce sync.Once
+
+// Probe ZMM_Hi256 offset of the current CPU.  Beware that core dumps may be
```

Comment:
This comment needs to start with AMD64XstateZMMHi256Offset.

Reply:
Fixed